avaframe/com1DFA/com1DFA.py

```diff
@@ -1370,9 +1391,47 @@ def DFAIterate(cfg, particles, fields, dem):
         infoDict.update({'stopInfo': {'Stop criterion': '< %.2f percent of PKE' % stopCritPer,
                                       'Avalanche run time [s]': '%.2f' % avaTime}})
 
+    ######## create range time diagram #####################
+    # export data for range-time diagram
+    if cfg['VISUALISATION'].getboolean('createRangeTimeDiagram'):
```

Comment:
is cfg['VISUALISATION'] the good place for createRangeTimeDiagram? or should we have another section (I dont know how to call it) with createRangeTimeDiagram and createAvalanchPath?

Reply:
good point, maybe tackle this in a future PR? when we came up with a good name